profile.md

```diff
@@ -350,6 +434,25 @@ deployment:
   "space_guid": "15823972-c216-4ba5-9f3f-e75b0b891297"
   ```
 
+- `app_annotations`
+
+  Version: 2.15
+
+  The [annotations](http://v3-apidocs.cloudfoundry.org/version/3.69.0/index.html#metadata) attached to the application that a Service Binding is associated with.
+  Note that the annotations of an application in Cloud Foundry MAY be changed.
+  This OPTIONAL property holds an object with the annotations key/value pairs. If present, this property MUST be an object, with one or more properties as follows:
```

Comment:
I’m writing a user story for this right now, and I’ve discovered an interesting scenario: when you want to remove all annotations (e.g. make public annotations -> private), then there is a problem with wording:

> If present, this property MUST be an object, with one or more properties

Because the only way I can delete all annotations is to send an empty object, and according to the line above that wouldn’t be valid.

@gberche-orange What do you think? should we fix that to `zero or more`?

Reply:
@waterlink thanks, good catch. I agree allowing for empty content will enable clearing previously set annotations during an update. I'll push new commits for fixing this.

Reply:
resolved in https://github.com/openservicebrokerapi/servicebroker/pull/658/commits/80268a5fc4d73d3778cb80424424cad8e9cd64a5